README.md

```diff
@@ -1,46 +1,58 @@
-# Roda project combine with Vue.js using webpack
+# Demonstration Todo Web App Combining Roda + Vue.js w/ Webpack
 
-This is a small project to combine Roda and Vue.js webpack
-We implement a simple allow you to add a new todo the todo list using sequel and sqlite3 in ruby
+This is a small project to demonstrate how to combine Roda and Vue.js with webpack.
+Running the application allows you to add/delete a todos the todo list.
 
-## How to set-up the project locally
+## Setting up the project locally
 
 1. set up the frontend module
 
-```
-$ npm i
-```
+  ```shell
+  npm i
+  ```
 
 2. build the frontend module
 
-```
-$ npm run prod
-```
+  ```shell
+  npm run prod
+  ```
 
 3. set the backend
-```
-$ bundle install
-```
+
+  ```shell
+  bundle config set --local without 'production'
+  bundle install
+  ```
 
 4. set the envirnment
-- rename the `./config/secrets_exapmle.yml` to `./config/secrets.yml`
+
+  Rename the `./config/secrets_exapmle.yml` to `./config/secrets.yml`
```

Comment:
Rename the `./backend-_app/config/secrets_exapmle.yml` to `./backend-_app/config/secrets.yml`

Reply:
done!

Reply:
please see new commit